grid-proxy/pkg/client/grid_client_test.go

```diff
@@ -344,12 +343,11 @@ func testSuccess(t *testing.T, f ProxyFunc) {
 }
 
 func TestPrepareURL(t *testing.T) {
-	status := "st"
 	freeMRU := uint64(10)
 	farmIDs := []uint64{1, 2, 3}
 	dedicated := true
 	filter := types.NodeFilter{
-		Status:    &status,
+		Status:    []string{"st"},
```

Comment:
Can you please explain this test? I don't get what it does

Reply:
it validates the output of the `prepareUrl` function. by making sure the `got` url path/query - returned from the `prepareUrl` function - equals to the `want` url path/query - created from a literal string -

so having a filter
```go
types.NodeFilter{
    Status:    []string{"st"}
}
```
should output `?status=st` and so on

Reply:
it validates the output of the `prepareUrl` function. by making sure the `got` url path/query - returned from the `prepareUrl` function - equals to the `want` url path/query - created from a literal string -

so having a filter
```go
types.NodeFilter{
    Status:    []string{"st"}
}
```
should output `?status=st` and so on